ingestion/src/metadata/cli/docker.py

```diff
@@ -35,7 +35,8 @@ def start_docker(docker, start_time, file_path, skip_sample_data):
     logger.info("Ran docker compose for OpenMetadata successfully.")
     if not skip_sample_data:
         logger.info("Waiting for ingestion to complete..")
-        ingest_sample_data(docker)
+        wait_for_containers(docker)
```

Comment:
Noticed that sometimes we had issues bringing up the docker containers. Now we wait for ingestion and server before triggering the sample data